### DISCUSSION TITLE
custom field duplicate value

### DISCUSSION THREAD

**Discussion by Akaki-1991** [YetiForceCompany/YetiForceCRM#16771] [How to] [answered]
can you help me, in which directory i will save edited file? custom module?

https://github.com/YetiForceCompany/YetiForceCRM/blob/7ee300ca64f3c541bf9a81665d2b214788907965/modules/Accounts/handlers/DuplicateVatId.php#L18

**Comment by apcloic** [comment, marked as answer]
@UNITED1878
Your question is not clear, if you want to customize some file and avoid loosing your modifications when updating yetiforce, you may use autoloader.

https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/loader-and-autoloader-of-files

To make it work, xon't forget to enable this mechanism in config/performance.php
``
'LOAD_CUSTOM_FILES' => true,
``
Regards

**Reply by Akaki-1991** [reply]
> @UNITED1878 Your question is not clear, if you want to customize some file and avoid loosing your modifications when updating yetiforce, you may use autoloader.
> 
> https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/loader-and-autoloader-of-files
> 
> To make it work, xon't forget to enable this mechanism in config/performance.php `'LOAD_CUSTOM_FILES' => true,` Regards

hello, thanks to reply.

i have custom module and I want to make a unique field.
